crates/release_plz_core/src/command/release_pr.rs

```diff
@@ -415,6 +415,7 @@ async fn github_create_release_branch(
     release_branch: &str,
 ) -> anyhow::Result<()> {
     repository.push(release_branch)?;
+    // TODO @justbeyou would we like to create arguments for those hardcoded commit messages also?
```

Comment:
Yes, I think it's what we need to do, right?

Reply:
I think so, but I observed that the commit message and PR title had different sources for their messages and I was wondering if this should also be changed in this PR.

Anyway, I was just exploring the codebase and writing some comments for myself to familiarize with the codebase.

Reply:
I see :)

> I observed that the commit message and PR title had different sources for their messages and I was wondering if this should also be changed in this PR.

Oh, I see. Yeah, the PR name and the commit message should be the same. Feel free to do this refactoring in this pr or another one 👍